AddinFiles/main.jsl

```diff
@@ -1,7 +1,6 @@
 ﻿/* About: Get OSI PI Data
 
-Get data from PI Servers, requires prior set up of PI DAX, OLE-DB and ODBC
-and odbc connections.
+Get data from PI Web API.
```

Comment:
Could you mention the minimum required PI Web API version required?  Was there any other setup you had to do on your system to get this to work?

Reply:
As far as I can remember there was no setup required after installing the correct version of the API.